src/endpoint/s3/s3_bucket_logging.js

```diff
@@ -0,0 +1,123 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const dbg = require('../../util/debug_module')(__filename);
+const http_utils = require('../../util/http_utils');
+
+function is_bucket_logging_enabled(source_bucket, bucket_name) {
+
+    if (!source_bucket || !source_bucket.bucket_info.logging) {
+        dbg.info("Bucket logging is not configured for bucket : ", bucket_name);
+        return false;
+    }
+
+    const enable = check_logging_config_perm(source_bucket.bucket_info.logging);
+    return enable;
```

Comment:
This check is unnecessary now. we can add it later if we see a reason to. 

```suggestion
    return true;
```

Reply:
done